tests/PHPStan/Rules/DeadCode/data/multiple_unreachable.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @param 'foo' $foo
+ */
+function foo($foo)
```

Comment:
This file needs a namespace.

Reply:
namespace added https://github.com/phpstan/phpstan-src/commit/66596b45eb61df295e3db477ae99e4d45a48edff